From b79ee5009bdbdfdf30d0d303c4b1f67d49988dd8 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sun, 20 Nov 2022 20:20:49 +0800
Subject: [PATCH 26/56] Revert "drm:pl111: Add of_node_put() when breaking out
 of for_each_available_child_of_node()"

This reverts commit 5e25bfcd12d8e90324e27b1adc1fc7410d3664ee.
---
 drivers/gpu/drm/pl111/pl111_versatile.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/pl111/pl111_versatile.c b/drivers/gpu/drm/pl111/pl111_versatile.c
index 963a5d5e6987..bdd883f4f0da 100644
--- a/drivers/gpu/drm/pl111/pl111_versatile.c
+++ b/drivers/gpu/drm/pl111/pl111_versatile.c
@@ -402,7 +402,6 @@ static int pl111_vexpress_clcd_init(struct device *dev, struct device_node *np,
 		if (of_device_is_compatible(child, "arm,pl111")) {
 			has_coretile_clcd = true;
 			ct_clcd = child;
-			of_node_put(child);
 			break;
 		}
 		if (of_device_is_compatible(child, "arm,hdlcd")) {
-- 
2.25.1

